packages/cli-tool/src/add-ons/ui-framework/index.ts

```diff
@@ -1,23 +1,26 @@
 import { cli } from 'cli-ux';
 
 import setupBootstrap from './bootstrap';
+import setupTailwindCss from './tailwind-css';
 
 export const UI_FRAMEWORK_OPTIONS: { [key: string]: string } = {
   bootstrap: 'Bootstrap',
-  tailwind: 'Tailwind CSS',
+  tailwindCss: 'Tailwind CSS',
   none: 'None',
 };
 
 export const setUIFramework = async(
   appName: string,
   uiFramework: string,
```

Comment:
Maybe
```suggestion
  uiFramework: keyof typeof UI_FRAMEWORK_OPTIONS,
```

Reply:
It is a little bit more complex because the array key cannot be a custom type (either "string" or "number" basically). So we need to switch using `Map` types instead. 

While a little bit "wordier", I like the idea to have auto-completion and error check for the add-ons key 😇 

Feel free to review and comment: implemented in ee7e4a7.